vector/src/main/java/im/vector/app/features/spaces/explore/SpaceDirectoryFragment.kt

```diff
@@ -71,6 +90,15 @@ class SpaceDirectoryFragment @Inject constructor(
         viewModel.selectSubscribe(this, SpaceDirectoryState::canAddRooms) {
             invalidateOptionsMenu()
         }
+
+        views.spaceCard.matrixToCardMainButton.isVisible = false
+        views.spaceCard.matrixToCardSecondaryButton.isVisible = false
+        views.spaceCard.knownMember1.isVisible = false
+        views.spaceCard.knownMember2.isVisible = false
+        views.spaceCard.knownMember3.isVisible = false
+        views.spaceCard.knownMember4.isVisible = false
+        views.spaceCard.knownMember5.isVisible = false
+        views.spaceCard.peopleYouMayKnowText.isVisible = false
```

Comment:
Why not using `spaceCardHelper` here, to reset the views?

Reply:
done